board/config_0.txt

```diff
@@ -0,0 +1,29 @@
+# Please note that this is only a sample, we recommend you to change it to fit
```

Comment:
We don't really need this file, it's basically one from `buildroot/board/raspberrypi/`, so we might just copy it from there.